setup.cfg

```diff
@@ -28,33 +29,52 @@ classifiers=
 python_requires = >=3.6
 packages = find:
 include_package_data = True
-setup_requires = setuptools_scm
+setup_requires =
+  # PEP-518 encourages the use of pyproject.toml for defining
+  # build dependencies...they should be defined under build-system.requires
 install_requires =
+  # ought to mirror requirements/install.txt
   astropy >= 3.1
   colorama >= 0.3
+  matplotlib >= 3.2
   numpy >= 1.14
   scipy >= 0.19
   setuptools >= 41.2
 
 [options.extras_require]
-classes =
+# a weird "bug" since python 2 allows extras to depend on extras
+# for backwards compatibility, setuptools will keep this active
+# see https://github.com/pypa/setuptools/issues/1260#issuecomment-438187625
+extras =
+  # ought to mirror requirements/extras.txt
+  # for developers
+  setuptools_scm
+  # for plasmapy.classes
   h5py
-  matplotlib
-diagnostics =
-  matplotlib
-tests =
-  pytest >= 5.1
-theory =
+  # for plasmapy.theory
   mpmath
   lmfit
+  # for plasmapy.utils.pytest_helpers
+  pytest >= 5.1
```

Comment:
I thought we fixed that... pytest should just be a dependency within `[tests]`, I think.

Reply:
There are a few things in `plasmapy.utils` that require pytest at the moment, but these will be mostly or entirely removed by #728.  After then, pytest should only be needed within `plasmapy.tests` and not `plasmapy.utils`.